docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -716,27 +692,41 @@ Date: Thu, 23 Aug 2018 11:57:17 GMT
 
 [{"id":1,"name":"Secretlessly Fluffy"}]
 </pre>
-We expect the request to respond with a JSON array containing all the pets.
 
-There we have it. This application is communicating with a target service without managing any secrets.
+We should get a 200 response with a JSON array of the pets.
+
+That's it! 
+
+The application is connecting to a password-protected Postgres database **without
+any knowledge of the credentials**.
+
+## Rotate Database Credentials
```

Comment:
I sort of think we should remove this section entirely. when we wrote it we were emphasizing rotation as a feature of secretless, but we're not doing that now. wdyt? it would simplify things a bit. we could add more content hammering home the point that the app connects to secretless _without credentials_, bc secretless takes care of all of that - the main value prop.